### PR TITLE
MiniLM fix

### PR DESCRIPTION
**Pull request by I8dNLo**
MiniLM fix.
Now it uses official implementation and pooling from SM repo

### CODE REVIEW

File: fastembed/text/mini_lm_embedding.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+]
+
+
+class MiniLMOnnxEmbedding(OnnxTextEmbedding):
+    @classmethod
+    def _get_worker_class(cls) -> Type[TextEmbeddingWorker]:
+        return MiniLMEmbeddingWorker
+
+    @classmethod
+    def mean_pooling(self, model_output, attention_mask):
```
Comment (joein) [inline fastembed/text/mini_lm_embedding.py:31]: type hints

File: fastembed/text/mini_lm_embedding.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        embeddings = output.model_output
+        attn_mask = output.attention_mask
+        return normalize(self.mean_pooling(embeddings, attn_mask)).astype(np.float32)
+
+
+class MiniLMEmbeddingWorker(OnnxTextEmbeddingWorker):
+    def init_embedding(
+        self,
+        model_name: str,
+        cache_dir: str,
```
Comment (joein) [inline fastembed/text/mini_lm_embedding.py:64]: kwargs are missing

File: fastembed/text/mini_lm_embedding.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    ) -> OnnxTextEmbedding:
+        return MiniLMOnnxEmbedding(
+            model_name=model_name, cache_dir=cache_dir, threads=1
```
Comment (joein) [inline fastembed/text/mini_lm_embedding.py:67]: kwargs are missing

File: tests/test_text_onnx_embeddings.py [hunk 4]
```diff
@@ -26,17 +26,27 @@
     "BAAI/bge-large-en-v1.5-quantized": np.array(
         [0.03434538, 0.03316108, 0.02191251, -0.03713358, -0.01577825]
     ),
-    "sentence-transformers/all-MiniLM-L6-v2": np.array([0.0259, 0.0058, 0.0114, 0.0380, -0.0233]),
+    "sentence-transformers/all-MiniLM-L6-v2": np.array(
+        [0.0259, 0.0058, 0.0114, 0.0380, -0.0233]
```
Comment (joein) [inline tests/test_text_onnx_embeddings.py:30]: canonical values should be updated

**Review by joein — CHANGES_REQUESTED** [review]
It's better not to mix style refactoring + actual code changes in one PR, because it becomes more difficult to review.
It might be ok to combine them in one PR once the code changes are approved

**Review by joein — APPROVED** [review]

**Comment by michaelfeil** [comment]
Awesome! @I8dNLo

**Pull request merged**